client/src/components/Charts/DoughnutChart.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Doughnut } from 'react-chartjs-2';
+
+const DoughnutChart = ({ userMonthlyBills }) => {
+  const type = userMonthlyBills.map((bill) => bill.name);
+  const amount = userMonthlyBills.map((bill) => bill.amount);
```

Comment:
```suggestion
const DoughnutChart = ({ type, amount }) => {
```
**do the mapping in the parent component then pass the props**